tests/test_pretty_visualize.py

```diff
@@ -0,0 +1,258 @@
+            "1": "<Module:: math_operations>‎",
+            "2": "<Section::>‎",
+            "3": "<Intrinsic:: IMPLICIT NONE>‎",
+            "4": "<Subroutine:: add>‎",
+            "5": "<Section::>‎",
+            "6": "<VariableDeclaration:: x, y>‎",
+            "7": "<VariableDeclaration:: result>‎",
+            "8": "<Section::>‎",
+            "9": "<Assignment:: result = x + y>‎",
+            "10": "<Subroutine:: subtract>‎",
+            "11": "<Section::>‎",
+            "12": "<VariableDeclaration:: x, y>‎",
+            "13": "<VariableDeclaration:: result>‎",
+            "14": "<Section::>‎",
+            "15": "<Assignment:: result = x - y>‎",
+            "16": "<Subroutine:: multiply>‎",
+            "17": "<Section::>‎",
+            "18": "<VariableDeclaration:: x, y>‎",
+            "19": "<VariableDeclaration:: result>‎",
+            "20": "<Section::>‎",
+            "21": "<Assignment:: result = x*y>‎",
+        },
+        "connectivity_list": {
+            "0": ["1"],
+            "1": ["2", "4", "10", "16"],
+            "10": ["11", "14"],
+            "11": ["12", "13"],
+            "14": ["15"],
+            "16": ["17", "20"],
+            "17": ["18", "19"],
+            "2": ["3"],
+            "20": ["21"],
+            "4": ["5", "8"],
+            "5": ["6", "7"],
+            "8": ["9"],
+        },
+    },
+    "trivial_fortran_files/nested_if_else_statements_subroutine.f90": {
+        "node_count": 12,
+        "edge_count": 11,
+        "node_labels": {
+            "0": "<Section::>‎",
+            "1": "<Subroutine:: nested_if_example>‎",
+            "2": "<Section::>‎",
+            "3": "<VariableDeclaration:: x, y>‎",
+            "4": "<Section::>‎",
+            "5": "x > 0‎",
+            "6": "y > 0‎",
+            "7": "<Intrinsic:: PRINT *, Bot...>‎",
+            "8": "<Intrinsic:: PRINT *, x i...>‎",
+            "9": "y > 0‎",
+            "10": "<Intrinsic:: PRINT *, x i...>‎",
+            "11": "<Intrinsic:: PRINT *, Bot...>‎",
+        },
+        "connectivity_list": {
+            "0": ["1"],
+            "1": ["2", "4"],
+            "2": ["3"],
+            "4": ["5"],
+            "5": ["6", "9"],
+            "6": ["7", "8"],
+            "9": ["10", "11"],
+        },
+    },
+}
+
+
+def get_property(node_edge_info, name):
+    for node_info, edge_info in node_edge_info:
+        if name in node_info and name in edge_info:
+            yield (node_info[name], edge_info[name])
+            continue
+
+        if name in node_info:
+            yield (node_info[name], None)
+            continue
+
+        if name in edge_info:
+            yield (None, edge_info[name])
+            continue
+
+        if node_info and edge_info:
+            raise KeyError(f"Keyword {name} not found!")
+
+
+@pytest.mark.skipif(not graphviz_present(), reason="Graphviz is not installed")
+@pytest.mark.parametrize("test_file", test_files)
+def test_Visualizer(here, test_file):
```

Comment:
```suggestion
def test_visualizer(here, test_file):
```

naming convention for functions is snake_case

Reply:
alright 65696625354a8904af110a250bccd5a3585eb9a7